.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ orbs:
 executors:
   linux-node:
     docker:
-      - image: cimg/node:18.20.3
```

Comment:
does this change the node version we do all of our testing on? I guess that's managed separately, right? or can we unify them now?

Reply:
No, this is only used for linting.   We do node compat testing via explicit download of node.

Reply:
emsdk uses 20.18 but sadly only 20.15 is available as a cimg on circleci.